presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Aggregation IF to FILTER by not unwrapping the IF expression instead Aggregation.
```

Comment:
@yuanzhanhku  I don't understand what this means exactly.
`by not unwrapping the IF expression instead aggregation`?

Isn't that the entire optimization?

Reply:
Sorry. There was a typo.
* Fix the Aggregation IF to FILTER rule by not unwrapping the IF expression inside the Aggregation.  The rule now rewrites `AGG(IF(condition, expr))` to `AGG(IF(condition, expr)) FILTER (WHERE condition)`.